src/scenarios/shared/startup.py

```diff
@@ -63,7 +94,7 @@ def runtests(self, traits: TestTraits):
         for key in ['apptorun', 'startupmetric', 'guiapp']:
             if not getattr(traits, key):
                 raise Exception('startup tests require %s' % key)
-        reportjson = os.path.join(TRACEDIR, 'perf-lab-report.json')
```

Comment:
Why did we get rid of this line?

Reply:
It was moved out of this specific case so other portions also have access to the reportjson variable.